people_tool/static/people_tool/js/directives/BadgeDirective.js

```diff
@@ -0,0 +1,38 @@
+(function () {
+    var app = angular.module('PeopleTool');
+
+    app.directive('badge', function() {
```

Comment:
This `function()` could be imported from elsewhere, I think, to avoid duplication with the majority of the course_info BadgeDirective.js -- but the lines above this calling out the specific app can't be modularized as easily.